test/cypress/features/accounts.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Accounts
+
+  Background:
+    Given Network switcher is enabled
+    And Network is set to devnet
```

Comment:
@eniolam1000752 Network switcher fails to change the network while the test is being run and that leads to the failure of the third test scenario

Reply:
It actually does on my machine and even on jenkins